02_Development/Display/features/environment.py

```diff
@@ -0,0 +1,8 @@
+
+
+def after_step(context, step):
+    if step.status == "failed":
+        try:
+            context.client.containers.get(context.container_name).kill()
+        except NameError:
```

Comment:
you want to print out the exception here, NameError or not, rather than just pass it.  If it's a NameError and that's common, then you can swallow the error (if you comment that in the code), but some other exception may happen, and you want to track that